backend/hct_mis_api/apps/payment/notifications.py

```diff
@@ -123,35 +101,48 @@ def _prepare_user_recipients(self) -> QuerySet[User]:
     def _prepare_action_user_and_date(self) -> None:
         PaymentNotification.ACTION_GET_DATA_MAP[self.action](self)
 
-    def _prepare_emails(self) -> List[EmailMultiAlternatives]:
+    def _prepare_emails(self) -> List[MailjetClient]:
         return [self._prepare_email(user) for user in self.user_recipients.exclude(id=self.action_user.id)]
 
-    def _prepare_email(self, user_recipient: User) -> EmailMultiAlternatives:
-        text_body, html_body = self._prepare_bodies(user_recipient)
-        email = EmailMultiAlternatives(
+    def _prepare_email(self, user_recipient: User) -> MailjetClient:
+        body_variables = self._prepare_body_variables(user_recipient)
+        email = MailjetClient(
+            mailjet_template_id=config.MAILJET_TEMPLATE_PAYMENT_PLAN_NOTIFICATION,
             subject=self.email_subject,
-            from_email=settings.EMAIL_HOST_USER,
-            to=[user_recipient.email],
-            body=text_body,
-            cc=[self.action_user.email],
+            recipients=[user_recipient.email],
+            ccs=[self.action_user.email],
+            variables=body_variables,
         )
-        email.attach_alternative(html_body, "text/html")
         return email
 
     def send_email_notification(self) -> None:
         if not config.SEND_PAYMENT_PLANS_NOTIFICATION or not self.enable_email_notification:
             return
         try:
             for email in self.emails:
-                email.send()
+                email.send_email()
         except Exception as e:
             logger.exception(e)
 
-    def _prepare_bodies(self, user_recipient: User) -> Tuple[str, str]:
-        context = self._prepare_default_context(user_recipient)
-        text_body = render_to_string("payment/payment_plan_action.html", context)
-        html_body = render_to_string("payment/payment_plan_action.txt", context)
-        return text_body, html_body
+    def _prepare_body_variables(self, user_recipient: User) -> Dict[str, Any]:
+        protocol = "https" if settings.SOCIAL_AUTH_REDIRECT_IS_HTTPS else "http"
```

Comment:
maybe we could use `EMAIL_USE_TLS` for that

Reply:
protocol is used here only for assembling a text of the link to the object, not email related, so I think it's fine with the current var?